build_packages.py

```diff
@@ -0,0 +1,61 @@
+import shutil
+import argparse
+import tempfile
+import os
+
+
+class TmpDir:
+    def __init__(self):
+        self.path = None
+
+    def __enter__(self):
+        self.path = tempfile.mkdtemp()
+        return self.path
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if self.path:
+            shutil.rmtree(self.path)
+
+
+if __name__ == "__main__":
+
+    TexTextVersion = "0.8.1"
+
+    parser = argparse.ArgumentParser(description="Build TexText distribution archive for all supported platforms.")
+    available_formats = [fmt for fmt, desc in shutil.get_archive_formats()]
+    parser.add_argument('--linux',
```

Comment:
Set of desired formats differs for different platforms. Linux should have tgz and zip, while for windows it's only zip. 
So currently script should be run twice, once for each platform to build only desired formats.

If you change options to indicate which formats should be build for each platform the script can be run once. Example: 
```bash
# build all at once
build_packages.py --linux-formats tgz zip --windows-formats zip 
# build windows only 
build_packages.py --windows-formats zip 
```

Reply:
I would just leave the Windows tgz on the disk :smile: 

But I will improve the command line options in the suggested way.

Reply:
I updated in that way. Not a "perfect" solution but I think it is enough.